acoustid/future/api/app.py

```
@@ -0,0 +1,89 @@
+
+        if len(fp.hashes) < 10:
+            raise ValidationError("Fingerprint too short")
+
+        print(simhash(fp.hashes))
+
+        print(fp)
+
+    return MsgspecResponse(content=SubmissionResponse(submission_id=1))
+
+
+async def handle_validation_error(request: Request, exc: Exception) -> MsgspecResponse:
```

Ruff (0.8.2)

Unused function argument: `request`

(ARG001)